src/main/resources/blockchain/default-genesis.json

```diff
@@ -0,0 +1,8905 @@
+{
+  "difficulty": "0x0400000000",
+  "extraData": "0x11bbe8db4e347b4e8c937c1c8370e4b5ed33adb3db69cbdb7a38e1e50b1b82fa",
+  "gasLimit": "0x1388",
+  "nonce": "0x0000000000000042",
+  "ommersHash": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+  "timestamp": "0x0",
+  "coinbase": "0x0000000000000000000000000000000000000000",
+  "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
It's a hash, so it's a number. It is something you compute by applying arithmetic transformations. Yes, it is a series 256 bits, because it's a representation of this number. Same as an `Int` is a series of 32 bits, and it is a number.

> They can also represent in image, you can interpret them however you want, still these are just bits.

No, it's either a number or an image, you can _represent_ them however you want. And if you're representation requires all digits to be specified, then `00` is not different from `0000` - they are both illegal.